modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

```diff
@@ -100,7 +100,7 @@ Quorum feature is only valid for components that use Object Store.
 
 === Object Store Persistence
 
-You can persistently store JDBC data in a centralized system, accessible from all cluster nodes, unless you are using Anypoint Runtime Fabric. Object Store Persistence is not currently supported for Runtime Fabric clusters. 
+You can persistently store JDBC data in a centralized system, accessible from all cluster nodes when using standalone runtimes. Object store persistence is not currently supported for Mule runtimes running on Runtime Fabric. 
```

Comment:
when using standalone runtimes > when running Mule runtime engine on premises. 
for Mule runtimes running on Runtime Fabric > on Runtime Fabric deployments.